README.md

```diff
@@ -19,6 +19,16 @@ Available variables are listed below, along with default values (see `defaults/m
 
 The `docker_edition` should be either `ce` (Community Edition) or `ee` (Enterprise Edition). You can also specify a specific version of Docker to install using the distribution-specific format: Red Hat/CentOS: `docker-{{ docker_edition }}-<VERSION>`; Debian/Ubuntu: `docker-{{ docker_edition }}=<VERSION>`.
 
+You can customize `daemon configuration` by providing `daemon.json` values in yaml format. Example:
```

Comment:
For my README files, it has the variable first, then the description. So this whole block should be after the "You can control..." paragraph, and the "You can customize..." line needs to be after the variable example. Also, the default should be used here (what is in `defaults/main.yml`).

Reply:
This line should also link to the daemon configuration documentation: https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file#daemon-configuration-file